source/dotnet/Library/AdaptiveCards.Rendering.Wpf/AdaptiveCardRenderer.cs

```diff
@@ -26,6 +26,7 @@ protected override AdaptiveSchemaVersion GetSupportedSchemaVersion()
         public AdaptiveCardRenderer(AdaptiveHostConfig hostConfig)
         {
             HostConfig = hostConfig;
+            FeatureRegistration.Set("acTest", "1.0");
```

Comment:
> FeatureRegistration.Set("acTest", "1.0"); [](start = 11, length = 42)

Wait is this test code or prod?

Reply:
oops -- good catch! this was supposed to be in the sample.

---
In reply to: [278357075](https://github.com/Microsoft/AdaptiveCards/pull/2560#discussion_r278357075) [](ancestors = 278357075)